packages/angular/projects/ui-angular/src/index.ts

```diff
@@ -21,8 +19,6 @@ export * from './lib/primitives';
 
 export * from './lib/primitives-poc/button';
 export * from './lib/primitives-poc/view';
-export * from './lib/primitives-poc/primitive/primitive.module';
-export * from './lib/primitives-poc/primitive/primitivie-component/primitive.component';
```

Comment:
This doesn't need to be exported at the top-level, this is for internal purposes.

Reply:
Yes, I just added that to check if it is working after adding that.